### PR TITLE
Fix the Tuning issue of helloworld tf_example2

### PR DESCRIPTION
**Pull request by zehao-intel**
## Type of Change

feature or bug fix or documentation or validation or others  
API changed or not: No

## Description

detail description 
JIRA ticket: [ILITV-2777](https://jira.devtools.intel.com/browse/ILITV-2777)

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR : Fixing

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed: No

### CODE REVIEW

**Review by lvliang-intel — APPROVED** [review]

**Comment by chensuyue** [comment]
Please remove the similar mnist example under keras example, we keep one is enough.

**Comment by chensuyue** [comment]
1. Please update helloworld2 readme and requirements, it doesn't match your code. 
2. Does your change impact Keras models? Should we have extension test?

File: examples/helloworld/tf_example2/README.md [hunk 1]
```diff
@@ -77,9 +77,9 @@ This customized metric will calculate accuracy.
 ```
 ### 3. Use the customized data loader and metric for quantization 
 ```python
-    dataset = Dataset()
-    dataloader = DataLoader(framework='tensorflow', dataset=dataset, batch_size=1)
-    config = PostTrainingQuantConfig()
+dataset = Dataset()
```
Comment (chensuyue) [inline examples/helloworld/tf_example2/README.md:80]: layout issue.
